GREEDY/DataManagers/IGraphManager.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+using GREEDY.Models;
+
+namespace GREEDY.DataManagers
+{
+    public interface IGraphManager
+    {
+        List<GraphData> GetDataForGraphs(string username, int? time);
```

Comment:
It makes the int able to be null ( for example if I write: int manoSkaicius = null; the compiler will show an error because an int can't be null. if I write: int? manoSkaicius = null; everything will be fine)

Reply:
nice. thanks